internal/compiler/engine.go

```diff
@@ -17,6 +17,13 @@ const (
 	ISODateFormat = "2006-01-02"
 
 	ISOTimeFormat = time.RFC3339
+
+	QueryFileName = "query.sql"
+
+	LoadMethod        = "LOAD_METHOD"
+	LoadMethodReplace = "REPLACE"
+
+	QueryFileReplaceBreakMarker = "\n--*--optimus-break-marker--*--\n"
```

Comment:
These properties do not seem to be used.

Reply:
removed